mmv1/products/iap/Settings.yaml

```diff
@@ -0,0 +1,289 @@
+            				   WorkforceIdentitySettings > GcipSettings/ If none is set,
+            				   default to use GOogle identity.
+          * `WORKFORCE_IDENTITY_FEDERATION`: Use external identities set up on Google Cloud Workforce
+            				     Identity Federation.
+        item_type: !ruby/object:Api::Type::Enum
+          name: 'identitySources'
+          description: |
+            Identity sources Enum
+          values:
+            - :IDENTITY_SOURCE_UNSPECIFIED
+            - :WORKFORCE_IDENTITY_FEDERATION
+  - !ruby/object:Api::Type::NestedObject
+    name: 'applicationSettings'
+    description: |
+      Top level wrapper for all application related settings in IAP.
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: 'csmSettings'
+        description: |
+          Settings to configure IAP's behavior for a service mesh.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'rctokenAud'
+            description: |
+              Audience claim set in the generated RCToken. This value is not validated by IAP.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'accessDeniedPageSettings'
+        description: |
+          Customization for Access Denied page. IAP allows customers to define a custom URI
+          to use as the error page when access is denied to users. If IAP prevents access
+          to this page, the default IAP error page will be displayed instead.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'accessDeniedPageUri'
+            description: |
+              The URI to be redirected to when access is denied.
+          - !ruby/object:Api::Type::Boolean
+            name: 'generateTroubleshootingUri'
+            description: |
+              Whether to generate a troubleshooting URL on access denied events to this application.
+          - !ruby/object:Api::Type::Boolean
+            name: 'remediationTokenGenerationEnabled'
+            description: |
+              Whether to generate remediation token on access denied events to this application.
+      - !ruby/object:Api::Type::String
+        name: 'cookieDomain'
+        description: |
+          The Domain value to set for cookies generated by IAP. This value is not validated by the API,
+          but will be ignored at runtime if invalid.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'attributePropagationSettings'
+        description: |
+          Settings to configure attribute propagation.
+        properties:
+          - !ruby/object:Api::Type::Array
+            name: 'outputCredentials'
+            description: |
+              Which output credentials attributes selected by the CEL expression should be propagated in.
+              All attributes will be fully duplicated in each selected output credential.
+              Possible values are:
+
+              * `OUTPUT_CREDENTIALS_UNSPECIFIED`: An output credential is required.
```

Comment:
`xxxx_UNSPECIFIED` is meaningless and can be removed.

Reply:
Sure, I will remove the unspecified values from enums

Reply:
Removed `XXXX_UNSPECIFIED`